npm-shrinkwrap.json

```diff
@@ -1,7395 +1,7258 @@
 {
   "name": "etcher",
   "version": "1.0.0",
+  "lockfileVersion": 1,
```

Comment:
That's already the new `package-lock.json` – it just gets renamed to `npm-shrinkwrap.json` if there's already one.